main.c

```diff
@@ -7,16 +7,78 @@
 #include "pak_get_file_type.h"
 #include "pak_header.h"
 #include "pak_pack.h"
+#include <stdio.h>
+
+void red () {

```

Comment:
Short functions could be marked as `inline` to give compiler a better hint to inline it.